packages/fiori/src/TimelineItem.js

```diff
@@ -88,6 +88,29 @@ const metadata = {
 			defaultValue: "-1",
 			noAttribute: true,
 		},
+
+		/**
+		 * Select orientation.
+		 *
+		 * @type {String}
+		 * @defaultvalue "vertical"
+		 * @public
+		 */
+		axisOrientation: {
+			type: String,
+			default:"vertical"
+		},
+
+		/**
+		 * Define the indicator line width.
+		 *
+		 * @type {Boolean}
+		 * @defaultvalue "vertical"
+		 * @public
```

Comment:
If this property is sets from the Timeline and not from app developer it should be private

Reply:
Done